src/__tests__/utils.compose-event-handlers.js

```diff
@@ -0,0 +1,45 @@
+import {composeEventHandlers} from '../utils'
+
+test('prevent default handlers when defaultPrevented is true', () => {
+  const handler = composeEventHandlers(
+    e => {
+      e.defaultPrevented = true
+    },
+    () => {
```

Comment:
Rather than this, could we use Jest mock functions (via `jest.fn()`) and assertions? Could make the `defaultCalled` cleaner as well.